frontend/src/plugins/extensions/routes.ts

```diff
@@ -0,0 +1,244 @@
+      path: '/',
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/external/*',
+      component: () => import('~/pages/external/ExternalRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/explore',
+      component: () => import('~/pages/exploreApplication/ExploreApplications'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/resources',
+      component: () => import('~/pages/learningCenter/LearningCenter'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/projects/*',
+      component: () => import('~/pages/projects/ProjectViewRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/notebookController/*',
+      component: () => import('~/pages/notebookController/NotebookController'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/notebook/:namespace/:notebookName/logout',
+      component: () => import('~/pages/notebookController/NotebookLogoutRedirect'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/modelServing/*',
+      component: () => import('~/pages/modelServing/ModelServingRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/modelCatalog/*',
+      component: () => import('~/pages/modelCatalog/ModelCatalogRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/modelRegistry/*',
+      component: () => import('~/pages/modelRegistry/ModelRegistryRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/pipelines/*',
+      component: () => import('~/pages/pipelines/GlobalPipelinesRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/pipelineRuns/*',
+      component: () => import('~/pages/pipelines/GlobalPipelineRunsRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/experiments/*',
+      component: () => import('~/pages/pipelines/GlobalPipelineExperimentsRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/artifacts/*',
+      component: () => import('~/pages/pipelines/GlobalArtifactsRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/executions/*',
+      component: () => import('~/pages/pipelines/GlobalPipelineExecutionsRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    flags: {
+      required: [SupportedArea.FINE_TUNING],
+    },
+    properties: {
+      path: '/modelCustomization/*',
+      component: () => import('~/pages/pipelines/GlobalModelCustomizationRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    properties: {
+      path: '/distributedWorkloads/*',
+      component: () => import('~/pages/distributedWorkloads/GlobalDistributedWorkloadsRoutes'),
+    },
+  },
+  {
+    type: 'app.route',
+    flags: {
+      required: [ADMIN_USER],
```

Comment:
FWIW, naturally we will follow up with proper work later, this has to go away in order to support future growth. BYO OIDC will cause some issues with the concept of "admin".

We'll be migrating away from "isAdmin" in the future in favour of SSAR checks